package/MDAnalysis/core/universe.py

```diff
@@ -238,6 +238,18 @@ class Universe(object):
     vdwradii: dict, ``None``, default ``None``
         For use with *guess_bonds*. Supply a dict giving a vdwradii for each
         atom type which are used in guessing bonds.
+    context: string or Guesser, default ``default``
+        Type of the Guesser to be used in attributes guessing
+    to_guess: list, (optional), defualt ``['types', 'masses']``
+              (in future versions types and masses will be removed)
+        Attributes to be guessed (these attributes will be either guessed
+        if they don't exist in the universe or partially guessed by only filling
+        its empty values if universe has the attribute)
```

Comment:
```suggestion
        TopologyAttrs to be guessed. These TopologyAttrs will be wholly guessed
        if they don't exist in the universe. If they already exist in the Universe,
        only empty or missing values will be guessed.
```

IMO the warning about future versions should be in a proper DeprecationWarning or similar, so I'm suggesting to remove it here. I'm also clarifying that it's TopologyAttrs here.